index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+      integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog=="
+      crossorigin="anonymous"
+    />
+    <title>Apple Cloned</title>
+  </head>
+  <body>
+    <nav class="main-nav d-flex absolute">
+      <ul class="d-flex space-b">
+        <li class="flex-item d-flex">
+          <a href="#"><i class="fab fa-apple"></i></a>
+        </li>
+        <li class="flex-item d-flex"><a href="#">Store</a></li>
+        <li class="flex-item d-flex"><a href="#">Mac</a></li>
+        <li class="flex-item d-flex"><a href="#">iPod</a></li>
+        <li class="flex-item d-flex"><a href="#">iPhone</a></li>
+        <li class="flex-item d-flex"><a href="#">iPad</a></li>
+        <li class="flex-item d-flex"><a href="#">iTunes</a></li>
+        <li class="flex-item d-flex"><a href="#">Support</a></li>
+      </ul>
+      <div class="search">
+        <i class="fas fa-search absolute"></i><input type="search" />
+      </div>
+    </nav>
+    <main class="container">
+      <div class="main-div w-1200px">
+        <a href="#"
+          ><div class="absolute">
+            <h1>What will your verse be?</h1>
+            <p>
+              Everyone has something to share. See how people are using iPad in
+              their unique ways to add to the world’s story.
+              <span
+                >Watch the “Your Verse” film and explore the stories
+                <i class="far fa-play-circle"></i
+              ></span>
+            </p></div
+        ></a>
+      </div>
+      <div class="section-below w-1200px">
+        <div class="box box1"></div>
+        <div class="box box2"></div>
+        <div class="box box3"></div>
+        <div class="box box4"></div>
```

Comment:
- [ ] They should be links, as is in the original.

![image](https://user-images.githubusercontent.com/61479648/91079703-c95bdb80-e61a-11ea-9364-0f900cca43ff.png)

- [ ] They also should cover the whole page (the width).

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91079871-09bb5980-e61b-11ea-8604-90e1b54c3e20.png) |![image](https://user-images.githubusercontent.com/61479648/91079896-150e8500-e61b-11ea-82f1-abcb0b55ae27.png)  |